com.unity.shadergraph/Editor/Data/Graphs/AbstractShaderProperty.cs

```diff
@@ -22,11 +22,19 @@ public bool gpuInstanced
             set {}
         }
 
-        internal virtual string GetHLSLVariableName(bool isSubgraphProperty)
+        internal virtual string GetHLSLVariableName(bool isSubgraphProperty, GenerationMode mode)
         {
+            if (mode == GenerationMode.VFX)
+                return $"PROP.{referenceName}";
```

Comment:
To support the GraphProperties struct that gets passed into the pixel/vertex functions, we need to modify the HLSL variable name to look up into that input struct.